### DISCUSSION TITLE
A more human-readable version of Annotated Folk Tales?

### DISCUSSION THREAD

**Discussion by pswoods** [j-hagedorn/trilogy#51] [General]
Hello. I found your project and used your dataset to make a hyperlinked ATU ↔ TMI. I used Google Sheets to lookup/filter/re-arrange the data and add formatting for my intended output; then I imported the new CSVs to Obsidian to make a sort of wiki in which all ATU, TMI, and Stories are linked. 

First of all, THANK YOU for making this available. I am a fiction writer (nearly code-illiterate) and this is just what I needed for my current project's research. 

My ask: do you have a less processed version of the _Annotated Folk Tales_ that you could share with me? Here's what I'm trying to do:
- The texts have no line breaks/paragraphs. I want the original paragraphs.
- I found one story (Ali Baba) that begins in medias res: "But the woman whom Ali Baba had married was poor and needy..." 
- I would love to have all the links that Ashliman has on his website (e.g. https://sites.pitt.edu/~dash/alibaba.html )
- Some ligatures/special characters are garbled. NBD - since they garble the same every time, I have been using GrepWin (regex replace) to fix them as I find them. 

Anyway, maybe you have something less cleaned-up but still linked via ATU? 

Thanks again,
Michael

**Comment by j-hagedorn** [comment]
Hey Michael,
Your transformation and use of the repo sounds compelling: I'd love to see the format of your wiki to see how the repo might be transformed into a useful format for authors using knowledge management tools like Obsidian or LogSeq.

Your request for a 'less processed' version of the AFT sounds like you might just want the raw output from Ashliman's site, before it was converted into structured data.  The removal of linebreaks and the exclusion of certain links and inconsistently formatted elements of the HTML were necessary in order to clean the data for analysis.

If you are willing to share your regex replace logic with me, I can apply it to the dataset for cleaning. 

Grateful to hear that this work has been of use to you!
